### PR TITLE
Added responseTime to LoggingFilter and Dashboard lookup performance fix

### PR DESCRIPTION
**Pull request by rvema**

### CODE REVIEW

File: api/src/main/java/com/capitalone/dashboard/service/DashboardServiceImpl.java [hunk 1]
```diff
@@ -144,7 +143,8 @@ public Dashboard get(ObjectId id) {
         List<ObjectId> collectorItemIds = collectorItems.stream().map(BaseModel::getId).collect(Collectors.toList());
         // Find the components that have these collector items
         List<com.capitalone.dashboard.model.Component> components = componentRepository.findByCollectorTypeAndItemIdIn(collectorType, collectorItemIds);
-        return dashboardRepository.findByApplicationComponentsIn(components);
```
Comment (tabladrum) [inline api/src/main/java/com/capitalone/dashboard/service/DashboardServiceImpl.java]: Can you make this a separate PR?

Comment (rvema) [reply api/src/main/java/com/capitalone/dashboard/service/DashboardServiceImpl.java]: Updated the description. Please review
